java/src/main/java/com/google/cloud/dataproc/templates/jdbc/JDBCToGCS.java

```diff
@@ -64,16 +71,48 @@ public JDBCToGCS() {
 
   @Override
   public void runTemplate() {
+    if (StringUtils.isAllBlank(jdbcSQL) && StringUtils.isAllBlank(jdbcSQLFile)) {
+      LOGGER.error(
+          "Required parameter not passed. Either {} or {} parameter must be provided. ",
+          JDBC_TO_GCS_SQL,
+          JDBC_TO_GCS_SQL_FILE);
+      throw new IllegalArgumentException(
+          "Required parameters for JDBCToGCS not passed. "
+              + "Set mandatory parameter for JDBCToGCS template "
+              + "in resources/conf/template.properties file or at runtime. "
+              + "Refer to jdbc/README.md for more instructions.");
+    } else if (StringUtils.isAllBlank(jdbcSQL)) {
```

Comment:
code should also throw an error when both jdbcSQL and jdbcSQLFile are populated.
i.e. it should ensure only one is provided. Also refer to GCSToJDBCConfig.java if all these validations can be done more neatly (somehow, I am not sure).

Reply:
Added JDBCToGCSConfig.java. Please review.

Reply:
@shashank-google Please review the changes.